server.py

```diff
@@ -76,9 +72,12 @@
 async def interface(request):
     async with ws_handler(request, ws_interfaces) as ws:
         # This message is sent only this (just connected) client
-        #await ws.send_json(robots[0], dumps=json.dumps)
+        # await ws.send_json(robots[0], dumps=json.dumps)
         await ws.send_json(state.as_dict(map_name), dumps=json.dumps)
-        #await ws.send_json(card_pack, dumps=json.dumps)
+        # print(state.as_dict(map_name))
+        dealt_cards = state.get_dealt_cards(assigned_robots[ws])
```

Comment:
tak tento řádek je na mě moc složitý a nechápu ho... hmmm už asi jo, dealt_cards bere jako argument toho robota a robota dostanu jako hodnotu při zadání tohoto klíče. dobrý :-D